src/lib/Preview.js

```diff
@@ -531,6 +535,11 @@ class Preview extends EventEmitter {
             /* eslint-disable no-console */
             console.error(`Error prefetching file ID ${fileId} - ${err}`);
             /* eslint-enable no-console */
+
+            const error = new Error(ERROR_CODE.prefetchFileId);
```

Comment:
MAybe I should abstract away, this functionality?

Reply:
@JustinHoldstock Maybe it would make sense to make a PreviewError which extends Error